webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path    = require("path")
+const webpack = require("webpack")
+
+module.exports = {
+  mode: "development",
+  devtool: "source-map",
```

Comment:
- [ ] I also noticed the mode is production whereas yours is development, again, kindly follow it to letters so that we wont have errors, anything can cause the error.